units/heroes/Brainless_Heroes.cfg

```diff
@@ -0,0 +1,222 @@
+#textdomain wesnoth-Danse_Macabre
+[unit_type]
+    id=Brainless Hero
+    [base_unit]
+        id=Brain Eater
+    [/base_unit]
+    advances_to=Battle Hero
+
+    # should be enabled in the end, since we
+    # a) don't want players in the beginning to see what heros will appear later
+    # b) nor do they have to see how the heroes look on higher levels (keeping some surprise)
+    # c) like the base_unit, they are shown in the help as advancements of the Soulless, which looks a bit odd
+    # d) the part without [variation] of this file is unused
+    # hide_help=yes
```

Comment:
But I did. Since then you only pushed 1 commit concerning Boreas' animations which (I believe) cannot solve his biggest problem.